recipe/requirements.txt

```diff
@@ -0,0 +1,2 @@
+codecov
+nose_warnings_filters
```

Comment:
Of course! In fact, looks like you have packaged codecov! You sandbagging us?
http://anaconda.org/jakirkham/codecov

nose_warnings_filters has not been packaged... but it doesn't look too deep... aside from using flit!